drivers/gpio/gpio_cc13xx_cc26xx.c

```diff
@@ -128,41 +131,38 @@ static int gpio_cc13xx_cc26xx_port_get_raw(const struct device *port,
 {
 	__ASSERT_NO_MSG(value != NULL);
 
-	*value = GPIO_readMultiDio(GPIO_DIO_ALL_MASK);
+	*value = HWREG(GPIO_BASE + GPIO_O_DIN31_0) & GPIO_DIO_ALL_MASK;
 
 	return 0;
 }
 
-static int gpio_cc13xx_cc26xx_port_set_masked_raw(const struct device *port,
-						  uint32_t mask,
-						  uint32_t value)
+static int gpio_cc13xx_cc26xx_port_set_bits_raw(const struct device *port, uint32_t mask)
 {
-	GPIO_setMultiDio(mask & value);
-	GPIO_clearMultiDio(mask & ~value);
+	HWREG(GPIO_BASE + GPIO_O_DOUTSET31_0) = mask;
```

Comment:
@Ayush1325 Is there any reason not to use the existing HAL functions rather than doing the low-level writes directly?

Reply:
Maybe I missed something but the new hal functions only seem to allow toggling individual pins instead of all the pins using mask.